docs/dashboard/apis.md

```diff
@@ -467,9 +467,386 @@ Responses
+|------------|--------|-------------------------| -------- |
+| name       | string | Unique Policy name      | Required |
+| policyType | string | Pass "WALLET_DEPLOYMENT" | Required |
+| policyData | object | Pass empty object (`{}`)  | Required |
+
+Responses
+
+> **_200 OK_**
+
+```json
+{
+    "statusCode": 200,
+    "message": "Paymaster Policy created successfully",
+    "data": {
+            "_id": "xxxxx-xxxx-xxxx-xxxx-xxxxx",
+            "name": "wallet deployment rule",
+            "policyType": "WALLET_DEPLOYMENT",
+            "organisationId": "xxxxx-xxxx-xxxx-xxxx-xxxxx",
+            "paymasterId": "xxxxx-xxxx-xxxx-xxxx-xxxxx",
+            "createdBy": "xxxxx-xxxx-xxxx-xxxx-xxxxx",
+            "chainId": 84532,
+            "active": true,
+            "policyData": {
+                "operator": "walletDeployment"
+             },
+       }     
+}
+```
+
+> **_401 Unauthorized_**
+
+```json
+{
+    "statusCode": 401,
+    "message": "Auth token and API key is required in the headers"
+}
+```
+
+
+
+#### 11. Get all rules for a paymaster (spending limit, webhook, wallet deployment, whitelisted contracts)
+
+> **_GET Request_**
+
+URL: https://paymaster-dashboard-backend.prod.biconomy.io/api/v2/public/sdk/paymaster-policy
+
+Parameters
+
+Header
+
+| Param     | Type   | Description                        | Required |
+| --------- | ------ | ---------------------------------- | -------- |
+| authToken | string | Token unique to every user account | Required |
+| apiKey    | string | API Key Associated with dApp       | Required |
+
+Responses
+
+> **_200 OK_**
+
+```json
+{
+  "statusCode": 200,
+  "message": "Paymaster Policies found!",
+  "data": [
+    {
+      "_id": "xxxxx-xxxx-xxxx-xxxx-xxxxx",
+      "name": "paymaster spending limit",
+      "policyType": "PAYMASTER_SA_LIMIT",
+      "organisationId": "xxxxx-xxxx-xxxx-xxxx-xxxxx",
+      "paymasterId": "xxxxx-xxxx-xxxx-xxxx-xxxxx",
+      "createdBy": "xxxxx-xxxx-xxxx-xxxx-xxxxx",
+      "chainId": 84532,
+      "active": true,
+      "policyData": {
+        "type": "PAYMASTER",
+        "startTimeInEpoch": 1709053430301,
+        "durationInMs": 10800000,
+        "cycleDuration": {
+          "value": 3,
+          "unit": "hour"
+        },
+        "threshold": 10,
+        "thresholdType": "COUNT"
+      }
+    },
+    {
+      "_id": "xxxxx-xxxx-xxxx-xxxx-xxxxx",
+      "name": "wallet deployment rule",
+      "policyType": "WALLET_DEPLOYMENT",
+      "organisationId": "xxxxx-xxxx-xxxx-xxxx-xxxxx",
+      "paymasterId": "xxxxx-xxxx-xxxx-xxxx-xxxxx",
+      "createdBy": "xxxxx-xxxx-xxxx-xxxx-xxxxx",
+      "chainId": 84532,
+      "active": true,
+      "policyData": {
+        "operator": "walletDeployment"
+      }
+    }
+  ]
+}
+```
+
+> **_401 Unauthorized_**
+
+```json
+{
+    "statusCode": 401,
+        "message": "Auth token and API key is required in the headers"
+}
+```
+
+#### 12. Update spending limit rule for a paymaster
```

Comment:
can you also specify the affect of this around cycle time reset.

Reply:
Done.,